scripts/run_etcd.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+mkdir -p external
+cd external
+
+git clone https://github.com/coreos/etcd
+cd etcd
+echo "Building etcd..."
+./build
```

Comment:
sure. i will merge this simple solution first. then i will do the git tricks to build the latest release.